beets/ui/commands.py

```diff
@@ -1666,6 +1666,7 @@ def print_completion(*args):
     u'/usr/share/local/bash-completion/bash_completion',
```

Comment:
Whoa, weird -- this does look wrong to me. I say this is probably a typo (influenced by the line above, which looks similar but is actually correct) and we should probably drop it in favor of @bismark's new line.